src/main/java/pl/zimi/zadanka/Zad6.java

```diff
@@ -1,5 +1,29 @@
 package pl.zimi.zadanka;
 
 public class Zad6 {
+    public int removeDuplicates(int[] nums) {
+        int Len = 0;
+        int numsLen = nums.length - 1;
+        int arrayRank = nums[numsLen];
 
+
+        int i = 0;
+        while (nums[arrayRank - 1] != arrayRank) {
+            if (i + 1 == nums.length) i = 0;
+            if (nums[i] - nums[i + 1] == 0) {
+                System.out.println("przerzut =" + "i[" + i + "] " + nums[i]);
+                for (int j = i + 1; j < numsLen; j++) {
+                    nums[j] = nums[j + 1];
+                }
+
+            }
+            i++;
+        }
+        for (int number : nums) {
```

Comment:
to nie powinno się znaleźć w tej metodzie
ta metoda jest żeby rozwiązać problem
a nie żeby drukować zawartość
to możesz wrzucić jako osobną metodę żeby drukowała zawartość